examples/simple.yaml

```diff
@@ -0,0 +1,59 @@
+esphome:
+  name: <device name>
+  friendly_name: <friendly name>
+  min_version: 2024.6.0
+  platformio_options:
+    # Run CPU at 160Mhz to fix mhi_ac_ctrl_core.loop error: -2
+    board_build.f_cpu: 160000000L
+
+esp8266:
+  board: d1_mini
+
+logger:
+  level: INFO
+  baud_rate: 0
+
+ota:
+  - platform: esphome
+    password: <pass here>
+
+wifi:
+  ssid: !secret wifi_ssid
+  password: !secret wifi_password
+
+  # Enable fallback hotspot (captive portal) in case wifi connection fails
+  ap:
+    ssid: <hotspot name>
+    password: <hotspot pass>
+
+captive_portal:
+
+api:
+  encryption:
+    key: <your key here>
+
+external_components:
+  - source: github://ginkage/MHI-AC-Ctrl-ESPHome@master
+    components: [ MhiAcCtrl ]
+
+# Version 4.0
+MhiAcCtrl:
+  # Only 20 (legacy) or 33 (includes 3D auto and vertical vanes) possible. 
+  # If you encounter mhi_ac_ctrl_core.loop error: -2 errors, change the frame_size to 20
+  frame_size: 33
+
+climate:
+  - platform: MhiAcCtrl
+    name: "MHI Air Conditioner"
+
+select:
+  - platform: MhiAcCtrl
+    vertical_vanes:
+      name: Fan Control Up Down
+    horizontal_vanes:
+      name: Fan Control Left Right
+
+switch:
+  - platform: MhiAcCtrl
+    vanes_3d_auto:
+      name: "3D Auto"
```

Comment:
Nitpick, the 'stop sign' here at the end actually means the previous line does not have a line ending.

So technically the last line does not have an ending, which might mess up some parsers and sometimes makes it harder to copy past.
(Just configure your favorite editor to always put a single line ending on save and never run into this again :wink: )